apps/greencheck/migrations/0021_add_created_to_greendomains.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 5.0.9 on 2025-06-12 16:59
+
+import django_mysql.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("greencheck", "0020_auto_20240223_2122"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="greendomain",
+            name="created",
+            field=models.DateTimeField(auto_now_add=True, null=True),
+            preserve_default=False,
+        ),
+        migrations.RunSQL(
+            sql="UPDATE greendomain SET created=modified",
```

Comment:
Hey @timcowlishaw  I've checked locally and it looks like with MariaDB, this migration will acquire a table level lock, when making this query.

On my machine, the migration locked the green domains table for nearly 2 minutes.

This is ok, it's better to be pre-warned when it happens.